dockers/docker-base/Dockerfile.j2

```diff
@@ -34,25 +34,45 @@ COPY ["etc/rsyslog.conf", "/etc/rsyslog.conf"]
 COPY ["etc/rsyslog.d/*", "/etc/rsyslog.d/"]
 COPY ["root/.vimrc", "/root/.vimrc"]
 
+# Install dependencies of supervisor
```

Comment:
>Install dependencies of supervisor [](start = 2, length = 34)

These packages should be defined as supervisor's runtime dependencies, following other sonic component Makefile.